browser/ui/tabs/brave_tab_menu_model.cc

```diff
@@ -85,7 +89,11 @@ std::u16string BraveTabMenuModel::GetLabelAt(size_t index) const {
   return TabMenuModel::GetLabelAt(index);
 }
 
-void BraveTabMenuModel::Build(int selected_tab_count) {
+void BraveTabMenuModel::Build(Browser* browser,
+                              TabStripModel* tab_strip_model,
+                              const std::vector<int>& indices) {
+  auto selected_tab_count = indices.size();
```

Comment:
nit: I think this is unused

Reply:
Actually, it's used in line 105 in the folded part below.

```
  auto toggle_tab_mute_label = l10n_util::GetPluralStringFUTF16(
      all_muted() ? IDS_TAB_CXMENU_SOUND_UNMUTE_TAB
                  : IDS_TAB_CXMENU_SOUND_MUTE_TAB,
      selected_tab_count);
```